simctrl/programmatic.rst

```diff
@@ -211,7 +211,21 @@ Functions
         :func:`fadvance` modifies NODED and NODERHS 
         but leaves NODEA and NODEB unchanged. 
 
-         
+----
+
+.. function:: nrnunit_use_legacy
+
+  Syntax:
+    ``h.nrnunit_use_legacy(0or1)``
+
+  Description:
+    | Arg of 0 uses modern codata2018 units for FARADAY, R, etc. (default as of version 8.0)
```

Comment:
Recommend suggesting `False` or `True` instead of `0` or `1`. (They're equivalent in NEURON, but I try to use  booleans where appropriate in the documentation.)

(I acknowledge that you can't do this with the environment variables, but I think that's okay.)

Reply:
Thanks.  And I'll specify that the return value is boolean as well. (I guess I need to mark it over in the c code).For environment variables, it's always possible, but tedious, to allow 1 or 0, Yes or No, On or Off, True or False, along with however much upper/lower case the user applies.